crates/admin/src/cluster_controller/scheduler.rs

```diff
@@ -764,15 +764,14 @@ mod tests {
 
         let partition_table =
             PartitionTable::with_equally_sized_partitions(Version::MIN, num_partitions);
-        let initial_scheduling_plan = SchedulingPlan::from(&partition_table, replication_strategy);
         let metadata_store_client = builder.metadata_store_client.clone();
 
         let networking = builder.networking.clone();
 
         let env = builder
-            .set_nodes_config(nodes_config.clone())
+            .set_nodes_config(node_ids[0].to_string(), node_ids[0], nodes_config.clone())
             .set_partition_table(partition_table.clone())
-            .set_scheduling_plan(initial_scheduling_plan)
+            .set_replication_strategy(replication_strategy)
```

Comment:
Yeah, I'm also not a fan of using "scheduling" here, but if we are going to change it, we need all occurrences at once, so perhaps this is better suited for a different PR. 

I'm not sure if replication is the best term here as well. That's because we have two places where we can use it; Bifrost's log replication property, and partition processor. 

Perhaps we can use "placement" for partition processors? As in "placement strategy" to set the algorithm, and "placement property" to define its properties. What do you think?